src/main/java/org/broadinstitute/hellbender/tools/spark/ParallelCopyGCSDirectoryIntoHDFSSpark.java

```diff
@@ -26,15 +26,18 @@
 import java.util.stream.Collectors;
 
 /**
- * Parallel copy a file or directory from GCS into HDFS.
+ * Parallel copy a file or directory from Google Cloud Storage into the HDFS file system used by Spark
```

Comment:
Yes, sorry but the omission of the period is for the oneLineSummary only. Here, within the JavaDoc, we'd like complete sentences. In fact, how we recapitulate the one line summary at the top of the JavaDoc actually bothers me because then the summary shows up twice in the forum tool doc,  once under the tool name and  once immediately under `Overview`:

![screenshot 2017-12-21 14 09 54](https://user-images.githubusercontent.com/11543866/34271054-ae5b1d2e-e658-11e7-8ecf-2ba1ac728fea.png)

However, this seems to be the entrenched practice and we are focused on the one-line summary and usage examples. If you feel generous, can we please have a period at the end of the javaDoc sentence. Same suggestion applies to docs that follow.